wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/css/openverse.css

```diff
@@ -1,8 +1,12 @@
+:root {
+    --header-height: 104.453px; /* redesiged global-header */
```

Comment:
Oh dear, it looks like the header is actually _taller_ 117px, on mobile. The breakpoint is `890px`.

Reply:
Good note, are there any other breakpoints?

Reply:
890 is the only breakpoint that changes the header height

Reply:
thanks!

Reply:
Updated in 4ce3c2e.